controllers/authController.js

```diff
@@ -13,38 +13,44 @@ const authService = require('../services/authService')
  */
 const githubAuth = (req, res, next) => {
   let userData
-  const authRedirectionUrl = `${config.get('services.rdsUi.baseUrl')}${config.get('services.rdsUi.routes.authRedirection')}`
+  const authRedirectionUrl = `${config.get(
```

Comment:
Can you remove all the unwanted code formatting from the PR?
Please submit PR with changes only in the code ehich id to be modified.
Seems your local code formatter is messing the code!